docs/migration/seidb_migration.md

```diff
@@ -135,7 +135,33 @@ pruning-interval = "1000"
 ```
 
 `PebbleDB` will be used as the default and recommended backend database for full node.
-If you wan to try out RocksDB, please follow [this guide](https://github.com/facebook/rocksdb/blob/master/INSTALL.md) to set up RocksDB environment.
+
+For RocksDB, follow these instructions to first install:
+```bash
+git clone https://github.com/facebook/rocksdb.git
+cd rocksdb
+
+DEBUG_LEVEL=0 make shared_lib install-shared
+
+export LD_LIBRARY_PATH=/usr/local/lib
+```
+If you run into any issues during installation, please refeference [this guide](https://github.com/facebook/rocksdb/blob/master/INSTALL.md).
+
+Once that is complete, you will need to add the following CGO flags:
+```bash
+CGO_CFLAGS="-I/path/to/rocksdb/include" CGO_LDFLAGS="-L/path/to/rocksdb"
+```
+
+and a `rocksdbBackend` tag:
+
+```bash
+-tags "rocksdbBackend"
+```
+
+to the seid go installation command.
```

Comment:
@philipsu522 @yzang2019 as mentioned, we can add more in the `Makefile` to install and setup rocksdb, but we have ran into a lot of issues here and not sure it is worth given it is not the default. Let me know your thoughts.